handler/oauth2/strategy_jwt.go

```diff
@@ -105,7 +140,21 @@ func (h *RS256JWTStrategy) generate(tokenType fosite.TokenType, requester fosite
 		return "", "", errors.New("GetTokenClaims() must not be nil")
 	} else {
 		claims := jwtSession.GetJWTClaims()
-		claims.ExpiresAt = jwtSession.GetExpiresAt(tokenType)
+
+		if claims.ExpiresAt.IsZero() {
+			claims.ExpiresAt = jwtSession.GetExpiresAt(tokenType)
```

Comment:
The idea here was to explicitly override the expires_at value. What's the reasoning to move it into `IsZero`?